allennlp/training/metrics/spearman_correlation.py

```diff
@@ -44,14 +44,14 @@ def __call__(
         # Flatten predictions, gold_labels, and mask. We calculate the Spearman correlation between
         # the vectors, since each element in the predictions and gold_labels tensor is assumed
         # to be a separate observation.
-        predictions = predictions.view(-1)
-        gold_labels = gold_labels.view(-1)
+        predictions = predictions.reshape(-1)

```

Comment:
For some reason, on GPU sometimes this `view` fails. We can use `reshape` anyway, that tries to do a `view` but if it can't will reshape.